package.json

```diff
@@ -65,7 +64,8 @@
     "release:publish:dry-run": "pnpm publish --recursive --tag latest --registry=\"http://localhost:4873/\"",
     "release:tag": "node scripts/releaseTag.mjs",
     "validate": "concurrently \"pnpm prettier && pnpm eslint\" \"pnpm proptypes\" \"pnpm docs:typescript:formatted\" \"pnpm docs:api\"",
-    "clean:node_modules": "rimraf --glob \"**/node_modules\""
+    "clean:node_modules": "rimraf --glob \"**/node_modules\"",
+    "build:charts:vendor": "node ./packages/x-charts-vendor/scripts/build.js"
```

Comment:
Is there a point in having this script if we can run `pnpm --filter @mui/x-charts-vendor build` or at least use it here? 🤔

Reply:
I was not aware of this `--filter` option